pkg/transport/kafka-client/kafka_producer_client.go

```diff
@@ -0,0 +1,118 @@
+package kafkaclient
+
+import (
+	"encoding/json"
+	"fmt"
+	"sync"
+
+	"github.com/confluentinc/confluent-kafka-go/kafka"
+	"github.com/go-logr/logr"
+	kclient "github.com/open-cluster-management/hub-of-hubs-kafka-transport/kafka-client/kafka-producer"
```

Comment:
why not call the create alias to the package as "producer"?
it's not so clear what is kclient.